code/app.py

```diff
@@ -331,20 +342,9 @@ def conversation_custom():
         chat_history = []
         for i, k in enumerate(user_assistant_messages):
             if i % 2 == 0:
-                chat_history.append(
-                    (
-                        user_assistant_messages[i]["content"],
-                        user_assistant_messages[i + 1]["content"],
-                    )
-                )
-        from utilities.helpers.ConfigHelper import ConfigHelper
-
-        messages = message_orchestrator.handle_message(
-            user_message=user_message,
-            chat_history=chat_history,
-            conversation_id=conversation_id,
-            orchestrator=ConfigHelper.get_active_config_or_default().orchestrator,
-        )
+                chat_history.append((user_assistant_messages[i]['content'],user_assistant_messages[i+1]['content']))
+        from backend.batch.utilities.helpers.ConfigHelper import ConfigHelper
+        messages = message_orchestrator.handle_message(user_message=user_message, chat_history=chat_history, conversation_id=conversation_id, orchestrator=ConfigHelper.get_active_config_or_default().orchestrator)
```

Comment:
Another line length change? (I think it's nice to avoid those in diffs unless it's due to an explicit style change, as it makes it harder to see what actually changed).

Reply:
Yes, I ignored these formatting changes during the last merge conflict and have now removed all line length related changes. Sorry for the inconvenience.